addons/resource.language.en_gb/resources/strings.po

```diff
@@ -6975,7 +6984,57 @@ msgctxt "#14110"
 msgid "Long date format"
 msgstr ""
 
-#empty strings from id 14111 to 15011
+#: Used in Confluence
+msgctxt "#14111"
+msgid "Activities"
+msgstr ""
+
+#: system/settings/settings.xml
+msgctxt "#14112"
+msgid "Enable activity logging"
+msgstr ""
+
+#: system/settings/settings.xml
+msgctxt "#14113"
+msgid "Enable notification activity logging"
+msgstr ""
+
+#: system/settings/settings.xml
+msgctxt "#14114"
+msgid "Show activity log"
+msgstr ""
+
+#: xbmc/activity/windows/GUIWindowActivityLog.cpp
+msgctxt "#14115"
+msgid "Basic"
```

Comment:
same here ofc, would be good to know how this label is used - in this case most likely:
#. Name of a notification/log severity level

Also, in this particular case we probably could reuse the according settings level label because they share a similar context? I'm fine with a separate label ofc if you think it's better.